.github/workflows/migrations-check.yml

```diff
@@ -15,8 +15,12 @@ jobs:
       matrix:
         os: [ ubuntu-20.04 ]
         python-version: [ 3.8 ]
+        django-version: ["pinned", "4.2"]
```

Comment:
A comment about what 'pinned' means would be helpful. Are we referring to what is currently pinned in base.in?

Reply:
`pinned` is used to install the latest patch version of Django within the global constraint i.e. Django==3.2.21 in current case because we have global constraint of Django<4.2.